src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,111 @@
+
+## The Modern PPL designer
+
+Now consider a PPL designer. PPL designers typically want to focus on model building and inference: graph representation, samplers, symbolic differentiation speed and stability...
+However for their PPL to be useful they must ensure users have easy access to diagnostics,
+visualizations, and model criticism tools.
+Suddenly the scope of their codebase grows, and duplicative functionality now exists across
+the open source Bayesian ecosystem
+
+## Life before az.InferenceData
+
+Before `az.InferenceData` there wasn't a consistent object or method
+
+- Each PPL designer would have to implement their own internal objects
+  - The objects may not be serializable to disk
+- All adjacent workflow libraries would only work with a subset of these objects, or would require users to restructure data each time.
+- The APIs and interfaces for all of these were inconsistent making life challenging for end users
+  - Folks using different PPLs would be isolated in completely different ecosystems, even when the PPLs share the same general programming language.
+
+## How az.InferenceData works and its implementation
+
+`az.InferenceData` is an object that stores (nearly) all the outputs of Bayesian Modeling in a consistent manner.
+Under the hood `az.InferenceData` is largely a collection of Xarray objects, with some utility functions built in as well.
+Xarray was the natural choice, because storing indexing, and performing calculations
+over multidimensional outputs are routine tasks for computational Bayesian modelers.
+
+![InferenceData Architecture](https://python.arviz.org/en/stable/_images/InferenceDataStructure.png)
+
+## How this enables ArviZ
+
+With a consistent object that represents the outputs of PPLs, it became possible to develop ArviZ,
+a unified library for exploratory analysis of Bayesian models.
+Now it didn't matter what PPL someone decided to use,
+as long as they follow the InferenceData specification ArviZ will know "what to do".
+This enables PPL designer to focus on the core of inference,
+and also ensures Bayesian Practitioners don't need to relearn the entire toolbox if they decide to switch PPLs.
+
+### The benefit of Xarray
+
+Xarray in turn simplifies the life of the ArviZ devs because we don't need to focus too much
+on the data store object, and instead we can focus on the statistical computations.
+Of the [key features of Xarray](https://xarray.dev/#features) we directly utilize
+
+- Interoperability
+- Operations over named dimensions
+- Value selection by label
+- Vectorized operations
+- Flexible and Extensible I/O backend API
+
+## Xarray Einstats
+
+The ArviZ project also maintains [xarray-einstats](https://einstats.python.arviz.org/en/latest/).
+It provides label aware wrappers for linear algebra (wrapping `numpy.linalg`), statistical computations like summary statistics, random sampling or working with distributions (wrapping `scipy.stats`), and reduce and rearrange (wrapping `einops`).
```

Comment:
The ArviZ project also maintains [xarray-einstats](https://einstats.python.arviz.org/en/latest/), which provides label-aware wrappers for the following functionalities:

- Linear algebra (wrapping `numpy.linalg`)
- Statistical computations like summary statistics, random sampling, or working with distributions (wrapping `scipy.stats`)
- Reduce and rearrange operations (wrapping `einops`)